src/peer/peer.js

```diff
@@ -118,96 +124,70 @@ class Peer {
    */
   async join(spaceName, threadName) {
     try {
-      const room = Peer._createRoomName(spaceName, threadName);
+      const roomName = utils.createRoomName(spaceName, threadName);
 
-      if (this.rooms[room] != null) {
-        this.rooms[room].lastUpdatedTime = Date.now(); // it's already attached, just update time
+      const room = this.state.rooms[roomName];
+      if (room != null) {
+        this.state.rooms.lastUpdatedTime = Date.now(); // it's already attached, just update time
```

Comment:
Shouldn't this be `this.state.rooms[roomName].lastUpdatedTime` rather than `this.state.rooms.lastUpdatedTime`?

Reply:
nice catch! it should be `room` 👍